scripts/travis/deploy_docker.sh

```diff
@@ -61,5 +61,9 @@ docker tag sqlflow:dev sqlflow/sqlflow:dev
 docker push sqlflow/sqlflow:dev
 
 echo "docker push sqlflow/sqlflow:$TRAVIS_TAG ..."
-docker tag sqlflow:ci sqlflow/sqlflow:$DOCKER_TAG
-docker push sqlflow/sqlflow:$DOCKER_TAG
+docker tag sqlflow:ci sqlflow/sqlflow:"$DOCKER_TAG"
+docker push sqlflow/sqlflow:"$DOCKER_TAG"
+
+echo "docker push sqlflow/sqlflow:mysql"
+docker tag sqlflow:mysql sqlflow/sqlflow:mysql
+docker push sqlflow/sqlflow:mysql
```

Comment:
In https://github.com/sql-machine-learning/sqlflow/blob/develop/scripts/travis/build.sh, we build sqlflow:dev and sqlflow:ci.  But I don't see where did we build sqlflow:mysql so could we push here?

Reply:
Oh, that's my bad, the `Deploy` stage running in a separated VM, we should re-build the MySQL docker image and push to Dockerhub.com